examples/documentation_examples/examples_test.go

```diff
@@ -45,6 +45,9 @@ func TestDocumentationExamples(t *testing.T) {
 	documentation_examples.ProjectionExamples(t, db)
 	documentation_examples.UpdateExamples(t, db)
 	documentation_examples.DeleteExamples(t, db)
+	documentation_examples.AggregationExamples(t, db)
```

Comment:
Judging from other examples in this file, you could conditionally skipping these by checking the server version with `testutil.CompareVersions`.

Reply:
I've moved the call to `AggregationExamples()` to a new function `TestAggregationExamples()` that will skip the example tests if the server version is less than `3.6`. Technically, it's only the fourth aggregation example that uses a `let` in a `$lookup`, but it felt odd to check the server version and skip only that example in `examples.go`. Let me know if I should do that instead of skipping all four examples here if the version is off.